app/packages/core/src/components/Grid/Grid.module.css

```
@@ -1,36 +1,25 @@
-@keyframes animatedBackground {
+/* Back layer: small, dark, slow */
+@keyframes fallingBack {
```

Stylelint (17.6.0)

[error] Expected keyframe name "fallingBack" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)